### DISCUSSION TITLE
Question - Which speed (air_speed, ground_speed, wind_speed) do we have to use for the acceleration issue ?

### DISCUSSION THREAD

**Discussion by acapeyron** [Estia-advanced-programming/pandora-public#124] [Announcements]
# What is you question about : 

- Have you check if a similar questions exists in the issues   ? 
- Have you check the update page on the wiki    ?  https://github.com/Estia-advanced-programming/pandora-public/wiki/Update

**Comment by WilliamDelamare** [comment]
We want the actual acceleration. Thus, use the ground speed.
